cargo-concordium/src/main.rs

```diff
@@ -134,10 +134,20 @@ struct Runner {
     #[structopt(
         name = "schema",
         long = "schema",
+        conflicts_with = "schema-v1",
         help = "Path to a file with a schema for parsing parameter or state (only for V0 \
                 contracts) in JSON."
     )]
     schema_path:         Option<PathBuf>,
+    #[structopt(
+        name = "schema-v1",
+        long = "schema-v1",
+        conflicts_with = "schema",
+        help = "Path to a file with a schema V1 file for parsing parameter or state (only for V0 \
+                contracts) in JSON. This is only needed if supplying an older version of the \
+                schema."
```

Comment:
I don't understand what you are saying here.

Schema V1 is for V0 contracts, and schema V2 is for V1 contracts. I thought we agreed we will keep supporting V2 schema for backwards compatibility reasons. People are using it.

Reply:
We are keeping backwards compatibility. The confusion here is whether the newest version of the schema is called version 1 or 2. The naming of the custom section is the main course for this.

Anyway, I have changed the help text and argument name, to use "unversioned" instead of V1, hopefully this is more clear